src/components/utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'http://localhost:9090/api',
+});
+
+export const getAllArticles = () => {
```

Comment:
this should replace the axios.get in view articles

Reply:
Done!